.travis.yml

```diff
@@ -0,0 +1,17 @@
+language: python
+python:
+- '2.7'
+- '3.4'
+- '3.5'
+- '3.6'
```

Comment:
Why are multiple versions defined here. Do you plan to add tests in the future with multiple versions?